tests/acceptance/stepDefinitions/notificationsContext.js

```diff
@@ -26,15 +27,22 @@ When('the user declines all shares displayed in the notifications on the webUI',
   return client.page.phoenixPage().declineAllSharesInNotification()
 })
 
-Given('app {string} has been enabled', function (app) {
+Given('app {string} has been {}', function (app, action) {
```

Comment:
```suggestion
Given('app {string} has been {string}', function (app, action) {
```

Reply:
It's not a string, it's phrased as 
`Given app "notifications" has been enabled` or `Given app "notifications" has been disabled`.